### ISSUE TITLE
onValueSelected is not working for me

### ISSUE THREAD

**Issue by kazanture** [appbaseio/reactivesearch#474]
**Issue Type:**
This(onValueSelected) is not working for me on IOS via expo. 

"@appbaseio/reactivesearch-native": "^0.9.0",

Any idea?

`      <ReactiveBase
        app=...
        credentials=...
      >
        <DataSearch
          fuzziness={1}
          debounce={500}
          componentId="searchbox"
          dataField=...
          autosuggest={true}
          placeholder=...
          onValueSelected={(value) => console.log('The selected value is', value)}
        />
      </ReactiveBase>`

**Comment by kazanture** [comment]
Ok I believe the change(onValueSelected for DataSearch component) is not pushed to master yet. Any idea when it will be pushed?

**Comment by divyanshu013** [comment]
Yeah, currently `onValueSelected` is only implemented on web. It might be a while before we can get to it on native. PRs are welcome too.
